NebulaWorld/Utils/UnityUtils.cs

```diff
@@ -0,0 +1,31 @@
+﻿using NebulaModel.Logger;
+using UnityEngine;
+
+namespace NebulaWorld.Utils
+{
+    public static class UnityUtils
```

Comment:
You should remove this and use the very useful plugin [RuntimeUnityEditor](https://github.com/ManlyMarco/RuntimeUnityEditor/releases) instead. Once installed, you only need to press F12 to open it in game and you will be able to inspect any GameObject, Components and it even have a runtime console to run custom code at runtime.

You just need to download it and put it in the BepInEx/Plugins folder next to where the Nebula plugin is.
![image](https://user-images.githubusercontent.com/69216913/110174287-e8a7a280-7dcd-11eb-98cf-daeb90c150cc.png)

Reply:
You are only telling me this NOW? I have been breaking my head trying to figure out whats going on inside the game and writing functions like this.

Reply:
Sorry, I added this info to the Wiki 😄